nuklear.h

```diff
@@ -389,7 +389,7 @@ extern "C" {
       #define NK_POINTER_TYPE unsigned __int32
     #elif defined(__GNUC__) || defined(__clang__)
       #if defined(__x86_64__) || defined(__ppc64__) || defined(__PPC64__) || defined(__aarch64__)
-        #define NK_POINTER_TYPE unsigned long
+        #define NK_POINTER_TYPE unsigned long long
```

Comment:
The changes in `nuklear.h` will have to be replicated in the `src` directory too.

Reply:
I'm unsure about changing this as part of the GDI demo updates. Going to recommend pushing that to a different change request.

``` suggestion
        #define NK_POINTER_TYPE unsigned long
```